spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -97,4 +99,24 @@
 	 */
 	Subscriber subscribe(String subscription, MessageReceiver messageHandler);
 
+	/**
+	 * Pulls and auto-acknowledges a number of messages from a Google Cloud Pub/Sub subscription.
+	 *
```

Comment:
These two lines of the JavaDoc on method level violate our Spring convention about JavaDocs: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting.

But let it be: we can address it in the separate issue.

Reply:
Thanks for pointing this out. I wasn't aware. Can we enforce these rules using checkstyle though?